content/guides/virtualizing/vmware-workstation.md

```diff
@@ -13,11 +13,11 @@ The required ISO file can be found at on the "Get Haiku" page of this website wh
 
 ##### Step 1. Downloading Haiku.
 
-Download the vmdk zip file from the Haiku website and unzip it. You should be able to see these following files. ![File system](/files/image1.jpeg)
+Download the vmdk zip file from the Haiku website and unzip it. You should be able to see these following files. ![File system](https://www.haiku-os.org/files/image1.jpeg)

```

Comment:
Uh, no, you need to add the files into the `static` directory in the proper subfolder, you can't just link to them on haiku-os.org.